lib/cog/commands/role.ex

```diff
@@ -1,18 +1,16 @@
 defmodule Cog.Commands.Role do
   @moduledoc """
-  This command allows the user to manage roles. Roles can contain users.
+  This command allows the user to manage roles.
 
   Usage:
       role --create <rolename>
       role --drop <rolename>
-      role --grant [--user=<username>|--group=<groupname>] <rolename>
-      role --revoke [--user=<username>|--group=<groupname>]  <rolename>
+      role --grant --group=<groupname> <rolename>
```

Comment:
We need to do those kind of changes across all the embedded commands, so it's something for later.